src/utils/helper.ts

```diff
@@ -8,5 +8,5 @@ export const formatAxiosError = <ErrorResponse = any>(
   if (!axios.isAxiosError(error)) {
     throw new AxiosError(String(error));
   }
-  throw error.response?.data || 'unknown axios error';
+  throw error.response?.data || new AxiosError('unknown axios error');
```

Comment:
Is `error.response?.data` an Error?

Reply:
error would be an AxiosError, not quite sure what error.response?.data is, just pulling this from https://github.com/PreventiaGroupLLC/ClientUI/blob/master/src/util/formatAxiosError.ts

Reply:
We should definitely only throw `Error` objects -- `error.response?.data` won't be that. :( I think the code you linked happens to be a bug.

This is unfair for late-stage feedback: could/should this be re-organized into an `axios` interceptor, so that we don't have to remember to call `formatAxiosError` everywhere?

Reply:
haha I originally had it as an axios interceptor and I can put it back as one

Reply:
Just to makes sure, we'll we add this to an axios interceptor in this PR or in a follow-up?